integration_tests/cli/auth-demo.sh

```diff
@@ -72,6 +78,13 @@ response_does_not_contain "$result" "\"address\": \"$user_address\""
 
 test_divider
 
+echo "Get list revoked account must be empty"
+result=$(dcld query auth list-revoked-account)
```

Comment:
Better to use `all-revoked-accounts` instead of `list-revoked-account` for the command to be consistent with other commands like `all-accounts`, `all-proposed-accounts-to-revoke`

Reply:
Done! ✅